src/modules/transfer/views/ViewTransferCreate.vue

```diff
@@ -602,6 +604,12 @@ export default {
       return 0;
     }
   };
+
+  const checkNaN = (number) => {
```

Comment:
what does this function do?

Reply:
It check the value in the current balance type, I put <div v-if="isNaN(sendXPX)"> which mean if the true that if will run the function to convert the NaN value to 0 instead of showing NaN value in the user interface.

Reply:
What are you trying to do ? The function will always return 0.

Reply:
![4a65a26e-8a93-4779-973c-875b3c51b628](https://user-images.githubusercontent.com/85566370/184658940-14adf8be-ed06-42ac-b4f7-abd9b5dbb5e0.png)
Yes, the function will always return 0. As you can see in the image, when the user did not input anything, the transfer amount and total will display as NaN value instead of 0.

Reply:
<div v-if= isNaN(xpxAmount) >0</div> 
Maybe this is a simpler solution?

Reply:
Oh, I never thought about putting 0 in there, I was so focus on convert NaN value to 0. Just now I try the simpler solution, yes, it work fine.

Reply:
Hi, I upload the code based on the suggestion solution. Please review it and inform me of any problems.